validator/src/main.rs

```diff
@@ -950,6 +954,32 @@ pub fn main() {
                 /* .default_value() intentionally not used here! */
                 .help("Keep this amount of shreds in root slots."),
         )
+        .arg(
+            Arg::with_name("shred_storage")
+                .hidden(true)
+                .long("shred-storage")
+                .value_name("SHRED_STORAGE")
+                .takes_value(true)
+                .possible_values(&["rocksdb", "rocksdb-fifo"])
+                .default_value("rocksdb")
+                .help("EXPERIMENTAL: The underlying ledger storage for shreds. \
+                       *WARNING*: You will lose your ledger data when you switch between options. \
+                       Possible values are: \
+                       'rocksdb': stores shreds using RocksDB's default (level) compaction. \
+                       'rocksdb-fifo': stores shreds under RocksDB's FIFO compaction. \
+                           This option is more efficient on disk-write-bytes of the ledger store."),
+        )
+        .arg(
+            Arg::with_name("shred_storage_size")
```

Comment:
looks like this arg is fifo specific?  if so it probably ought to include “fifo” or “rocksdb-fifo” in its name somewhere